lib/cache/index.js

```diff
@@ -15,8 +17,9 @@ module.exports = (strapi, type = 'mem', opts = {}) => {
 
   opts.max = opts.max || 500;
   opts.maxAge = opts.maxAge || 60 * 60 * 1000;
+  const { cacheTimeout = 500, ...storeOpts } = opts;
```

Comment:
I like the idea of a timeout 👍  
Could you add it to the list of options in the README ? Thanks !